src/frontend/Sider/Journalføring/Standard/AvsenderPanel.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react';
+
+import styled from 'styled-components';
+
+import { EnvelopeClosedFillIcon, EnvelopeClosedIcon } from '@navikt/aksel-icons';
+import {
+    Box,
+    Checkbox,
+    CopyButton,
+    ExpansionCard,
+    HStack,
+    Label,
+    TextField,
+} from '@navikt/ds-react';
+import { ABlue500 } from '@navikt/ds-tokens/dist/tokens';
+
+import { PanelHeader, PanelHeaderType } from './PanelHeader';
+import { JournalføringState } from '../../../hooks/useJournalføringState';
+import { JournalpostResponse } from '../../../typer/journalpost';
+
+const ExpansionCardHeader = styled(ExpansionCard.Header)`
+    padding-bottom: 0.35rem;
+`;
+
+const ExpansionCardContent = styled.div`
```

Comment:
Gjerne bruk `HStack` i stedet for div med flex, noe ala: 
```suggestion
const ExpansionCardContent = styled(HStack).attrs({ gap: '4'})`
```
kan erstatte linje 26-28

Reply:
fixd: [a871251](https://github.com/navikt/tilleggsstonader-sak-frontend/pull/260/commits/a87125147fc4a043e2cf32156759a2d22a6a1093)